src/submit_jobs.py

```diff
@@ -187,13 +223,22 @@ def read_job(job: SparkJob):
     app_name = f"iasworld_{current_datetime}"
     args = parse_args()
```

Comment:
**[Suggestion, non-blocking]** What do you think about adding a quick log line here to record all of the values set on `args`?

Reply:
Done in 612f03e!